apps/board/src/board/board.service.ts

```diff
@@ -1,32 +1,67 @@
 import { Injectable, NotFoundException } from '@nestjs/common';
 import { InjectRepository } from '@nestjs/typeorm';
+import { CareerCategory, PositionCategory } from '@publicData/entities';
 import { Repository } from 'typeorm';
 import { Transactional } from 'typeorm-transactional';
-import { CreateBoardDto } from './dto/create-board.dto';
-import { UpdateBoardDto } from './dto/update-board.dto';
+import { CreateBoardDto, UpdateBoardDto } from './dto/req.dto';
 import { Board } from './entities/board.entity';
 
 @Injectable()
 export class BoardService {
   public constructor(
     @InjectRepository(Board)
     private readonly boardRepository: Repository<Board>,
+    @InjectRepository(CareerCategory)
+    private readonly careerCategoryRepository: Repository<CareerCategory>,
+    @InjectRepository(PositionCategory)
+    private readonly positionCategoryRepository: Repository<PositionCategory>,
   ) {}
 
   @Transactional()
-  public async getPosts(): Promise<Board[]> {
-    const posts = await this.boardRepository.find();
+  public async getPostsByPostCategory(postCategoryId: number): Promise<Board[]> {
```

Comment:
여기까지 이렇게 되는게 적절하겠네요~
```suggestion
  public async getPostsByPostCategoryId(postCategoryId: number): Promise<Board[]> {
```

Reply:
감사합니다. 
수정된 커밋입니다 : 86f6e771c301ed5f9d9708ca677e8624051a0498